Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/formatter/FormatterStatistics.kt

```diff
@@ -0,0 +1,41 @@
+@file:JvmName("FormatterStatistics")
```

Comment:
is it needed? we don't have a Java code as far as I know it help when you to use the code from Java

Reply:
yes, as it's intended to be used with ViewBinding